.github/workflows/dist.yml

```diff
@@ -36,13 +36,6 @@ jobs:
           echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin
           export POLYNOTE_VERSION=${GITHUB_REF#refs/tags/}
           export SPARK_VERSION="2.4.5"
-          export SCALA_VERSION="2.11"
-          docker build --build-arg POLYNOTE_VERSION --build-arg SCALA_VERSION -t polynote/polynote:${POLYNOTE_VERSION}-${SCALA_VERSION} docker/base/
-          docker push polynote/polynote:${POLYNOTE_VERSION}-${SCALA_VERSION}
-          docker build --build-arg POLYNOTE_VERSION --build-arg SCALA_VERSION -t polynote/polynote:${POLYNOTE_VERSION}-${SCALA_VERSION}-spark2.4 docker/spark
-          docker push polynote/polynote:${POLYNOTE_VERSION}-${SCALA_VERSION}-spark2.4
-          docker tag polynote/polynote:${POLYNOTE_VERSION}-${SCALA_VERSION}-spark2.4 polynote/polynote:latest
-          docker push polynote/polynote:latest
```

Comment:
@jeremyrsmith Updated `dist.yml` like we discussed. I _think_ this is the only change necessary? I guess we will find out one way or another once this is merged...